### DISCUSSION TITLE
Markdown Table Support

### DISCUSSION THREAD

**Discussion by anaclumos** [notaapp/talk#316] [Feature Requests] [answered]
Markdown Table deserves better support!

<img width="780" alt="image" src="https://user-images.githubusercontent.com/31657298/192971027-45e60f86-2ca2-4efd-baa4-97b4c9fa5d56.png">

**Comment by astoilkov** [comment, marked as answer]
This is a duplicate. We are discussing the topic in #241 and #284.
